sys/universal_address/universal_address.c

```diff
@@ -294,6 +303,10 @@ void universal_address_init(void)
 {
     mutex_lock(&mtx_access);
 
+    /* cppcheck-suppress unsignedLessThanZero
+     * (reason: UNIVERSAL_ADDRESS_MAX_ENTRIES may be zero in which case this
+     * code is optimized out)
+     */
```

Comment:
Mhhhh this comments aren't supposed to be on the same line... Look at the diff please, so I don't spam your inbox ;-)